python/lsst/jointcal/utils.py

```diff
@@ -361,12 +365,12 @@ def plot_flux_distributions(plt, old_mag, new_mag, old_jitter, new_jitter,
     plt.savefig(filename.format(name))
 
 
-def plot_all_wcs_deltas(plt, data_refs, visit_catalogs, old_wcs_list, name,
+def plot_all_wcs_deltas(plt, data_refs, visits, old_wcs_list, name,
```

Comment:
Here and in the functions below, please define the parameters and return values.